pontoon/sync/formats/silme.py

```diff
@@ -78,6 +85,9 @@ def __init__(self, parser, path, source_resource=None):
         self.source_resource = source_resource
         self.entities = OrderedDict()  # Preserve entity order.
 
+        # Bug 1193860: unescape quotes in some files
+        self.escape_quotes_on = 'mobile/android/base' in path and parser is DTDParser
```

Comment:
Yeah, that was my original idea (do it for all .dtd files), but it's sadly not possible:
https://dxr.mozilla.org/mozilla-central/source/python/compare-locales/compare_locales/checks.py#389

We have to do it the ugly way. One day we should get rid of the hardcoded path and move this to project config.